tests/run.rs

```diff
@@ -1,30 +1,18 @@
+#![cfg_attr(
+    feature = "dev",
```

Comment:
i think this means that this file is only compiled when `--cfg feature="dev"` is passed to rustc... but what i'm not sure about is what you intend to consider "dev". is that for running tests locally? does it get run on CI? curious if there's a more descriptive feature name than "dev" that could make this clearer.

Reply:
No, this doesn't exclude the file from compilation. This says that if `--features dev` is passed to `cargo` that a few warnings are disabled. I use this during development to suppress certain warnings that I usually don't want to see while working on the code.

Reply:
ah i see, the feature flag applies to the args below. makes more sense then 👍